src/SessionSetup/SessionSetupUtilsTest.cpp

```diff
@@ -31,4 +32,48 @@ TEST(SessionSetupUtils, CredentialsFromSshInfoWorksCorrectly) {
   EXPECT_EQ(info.user.toStdString(), credentials.user);
 }
 
+const uint32_t kPid = 100;
+const char* kProcessName = "process_name";
+const char* kProcessPath = "/path/to/process_name";
+
+std::vector<orbit_grpc_protos::ProcessInfo> SetupTestProcessList() {
+  using orbit_grpc_protos::ProcessInfo;
+
+  auto expected_target_process = ProcessInfo();
+  expected_target_process.set_pid(kPid);
+  expected_target_process.set_name(kProcessName);
+  expected_target_process.set_full_path(kProcessPath);
+
+  auto lower_pid_process = ProcessInfo();
+  lower_pid_process.set_pid(kPid - 1);
+  lower_pid_process.set_name(kProcessName);
+  lower_pid_process.set_full_path(kProcessPath);
+
+  auto different_process = ProcessInfo();
+  different_process.set_pid(kPid + 1);
+  different_process.set_name("some_other_process");
+  different_process.set_full_path("/path/to/some_other_process");
+
+  return {different_process, lower_pid_process, expected_target_process, different_process,
+          lower_pid_process};
```

Comment:
Is it intended to return the same process twice?

Reply:
Yes. I wanted to catch several different cases and combinations of processes with different and equal name before and after the actual target process, and re-using the same processes made the test more concise. Do you think it's easier to understand if I actually define 5 different processes with different PIDs?

Reply:
(Both would be fine with me.)

Reply:
So... I did both, that's what you asked for, right? ;-)

Reply:
I'd love to answer with a meme now. :-)